create-image.py

```diff
@@ -66,7 +66,7 @@ def main():
     variant_info = VARIANTS[args.variant]
     if args.variant == 'gpu':
         variant_info = variant_info[args.cuda_version]
-    image_name = 'CC-Ubuntu{}{}'.format(version_number, variant_info['name-suffix'])
+    image_name = 'Sage{}{}'.format(version_number, variant_info['name-suffix'])
```

Comment:
It would be good to change the "name-suffix" on line 41 from "-SAGE" to "-VIRTUAL-WAGGLE". This would now output an image named "Sage-SAGE". Changing the name-suffix would update the name to "Sage-VIRTUAL-WAGGLE".